Indicators/AroonOscillator.cs

```diff
@@ -62,14 +62,14 @@ public AroonOscillator(int upPeriod, int downPeriod)
         public AroonOscillator(string name, int upPeriod, int downPeriod)
             : base(name)
         {
-            var max = new Maximum(name + "_Max", upPeriod);
+            var max = new Maximum(name + "_Max", upPeriod + 1);
             AroonUp = new FunctionalIndicator<IndicatorDataPoint>(name + "_AroonUp",
                 input => ComputeAroonUp(upPeriod, max, input), 
                 aroonUp => max.IsReady,
                 () => max.Reset()
                 );
 
-            var min = new Minimum(name + "_Min", downPeriod);
+            var min = new Minimum(name + "_Min", downPeriod + 1);
```

Comment:
do these changes permit the aroon tests to pass without the try/catch assertion code? If so we should clean that up.